kod/object/item/passitem/ring/ringwedd.kod

```diff
@@ -157,21 +157,6 @@ messages:
       return FALSE;
    }
 
-   % Things we handle to be a defense modifier:
-   ModifyDefensePower(who = $,what = $,defense_power = 0)
-   "Increase or decrease the defense of the defender."
-   {
-      % Add a small amount of defense.  Only added as a defmod if the user is
-      %  One of the original 2 users, and both user slots are filled.
-      return defense_power + viDefense_bonus;
```

Comment:
Removing this function orphans `viDefense_bonus` under `classvars`. I recommend removing that as well.